src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentsPageV2.jsx

```diff
@@ -0,0 +1,279 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Switch, Route, useHistory } from 'react-router-dom';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import recordEvent from 'platform/monitoring/record-event';
+
+import ScheduleNewAppointment from './ScheduleNewAppointment';
+import * as actions from '../../redux/actions';
+import CancelAppointmentModal from '../cancel/CancelAppointmentModal';
+import {
+  getCancelInfo,
+  selectFutureStatus,
+  selectExpressCareAvailability,
+} from '../../redux/selectors';
+import {
+  selectFeatureRequests,
+  selectFeaturePastAppointments,
+  selectFeatureDirectScheduling,
+  selectFeatureCommunityCare,
+  selectFeatureExpressCare,
+  selectIsWelcomeModalDismissed,
+  selectIsCernerOnlyPatient,
+  selectFeatureProjectCheetah,
+  selectFeatureHomepageRefresh,
+} from '../../../redux/selectors';
+import { GA_PREFIX, FETCH_STATUS } from '../../../utils/constants';
+import { scrollAndFocus } from '../../../utils/scrollAndFocus';
+import TabNav from './TabNav';
+import RequestExpressCare from './RequestExpressCare';
+import FutureAppointmentsListV2 from '../FutureAppointmentsListV2';
+import PastAppointmentsList from '../PastAppointmentsList';
+import PageLayout from './PageLayout';
+import DowntimeNotification, {
+  externalServices,
+} from 'platform/monitoring/DowntimeNotification';
+import WarningNotification from '../../../components/WarningNotification';
+import ScheduleNewProjectCheetah from './ScheduleNewProjectCheetah';
+import Select from './Select';
+
+const pageTitle = 'VA appointments';
+
+const options = [
+  { label: 'Upcoming', value: 'upcoming' },
+  { label: 'Requested', value: 'requested' },
+  { label: 'Past', value: 'past' },
+  { label: 'Cancelled', value: 'cancelled' },
+];
+
+function AppointmentsPageV2({
+  cancelInfo,
+  closeCancelAppointment,
+  confirmCancelAppointment,
+  expressCare,
+  fetchFutureAppointments,
+  fetchExpressCareWindows,
+  futureStatus,
+  isCernerOnlyPatient,
+  isWelcomeModalDismissed,
+  pendingStatus,
+  showCommunityCare,
+  showDirectScheduling,
+  showExpressCare,
+  showPastAppointments,
+  showScheduleButton,
+  showCheetahScheduleButton,
+  startNewAppointmentFlow,
+  startNewExpressCareFlow,
+  showHomePageRefresh,
+}) {
+  useEffect(() => {
+    document.title = `${pageTitle} | Veterans Affairs`;
+
+    if (showExpressCare && futureStatus === FETCH_STATUS.notStarted) {
```

Comment:
I think you can remove this `if` block. I believe we can fetch the appointments we need in the sub pages for each view in that dropdown.

Reply:
Done.